src/Clock.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  clockName: string,
+};
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(this.timer, 1000);
+  }
+
+  componentDidUpdate(prevState: Props) {
+    const { clockName } = prevState;
+
+    if (this.props.clockName !== clockName) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${clockName} to ${clockName}`);
+    }
```

Comment:
Clock-0 to Clock-0?
<img width="271" alt="Screenshot 2022-09-16 at 18 26 48" src="https://user-images.githubusercontent.com/47606379/190675057-25e5f829-044a-418f-bf02-aeedf7aed678.png">